packages/cli/src/options/beaconNodeOptions/chain.ts

```diff
@@ -8,9 +8,8 @@ export interface IChainArgs {
   "chain.persistInvalidSszObjects": boolean;
   "chain.proposerBoostEnabled": boolean;
   "chain.defaultFeeRecipient": string;
+  "chain.maxSkipSlots"?: number;
   "safe-slots-to-import-optimistically": number;
-  // this is defined as part of IBeaconPaths
-  // "chain.persistInvalidSszObjectsDir": string;
```

Comment:
Because I was not sure of the value of keeping the property commented out...it felt like something that was marked for removal. 

I have brought it back and updated it a bit to be more clear.